controllers/chat/ChatController.js

```diff
@@ -433,6 +435,7 @@ module.exports = {
     }
   },
   findOrCreateChannel: async (req, res) => {
+    console.log('findOrCreateChannel', req.body);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding a console log for debugging purposes is fine during development, but it's generally not a good practice to leave them in production code as they can expose sensitive information. Consider using a logging library that can be toggled for different environments or remove this if it's no longer necessary.
```diff
- console.log('findOrCreateChannel', req.body);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->